css/css.go

```diff
@@ -281,11 +283,15 @@ func (c *cssMinifier) minifyDeclaration(property []byte, components []css.Token)
 	simple := true
 	prevSep := true
 	values := c.valuesBuffer[:0]
+
+outerLoop:
 	for i := 0; i < len(components); i++ {
-		comp := components[i]
-		if comp.TokenType == css.LeftParenthesisToken || comp.TokenType == css.LeftBraceToken || comp.TokenType == css.LeftBracketToken || comp.TokenType == css.RightParenthesisToken || comp.TokenType == css.RightBraceToken || comp.TokenType == css.RightBracketToken {
+		comp := &components[i]
```

Comment:
Wasn't expecting any significant performance boost here. But simply taking the address of an existing object does mean copying just 8 bytes (a pointer) instead of 28 (a uint32 and the slice's two ints and pointer).

Reply:
It also means pointer indirection at all accesses, and I think it is equally likely this change will make it run slower. Unless performance measurements show an advantage, I would like to leave this out.

Reply:
Sounds great. Pushing up fix now.